libs/integration/src/icon/vl-all-icons.component.ts

```diff
@@ -0,0 +1,64 @@
+
+                return html`
+                    <div class="container">
+                        ${allIcons.map((icon) => html`
+                                <div class="icon">
+                                    <vl-icon-next icon=${icon}></vl-icon-next>
```

Comment:
Zou dat niet cool zijn (en vooral nuttig) als je op een icon klikt, dat je de template direct krijgt in je clipboard?

bv. `<vl-icon-next icon="alarm-silent"></vl-icon-next>`

Dat kan op zich gwn met bv. dit toe te voegen aan click handler:

```ts
 handleClick(icon: string) {
        const clipboardValue = `<vl-icon-next icon="${icon}"></vl-icon-next>`;
        navigator.clipboard.writeText(clipboardValue);
    }
```

Ook mogelijks zeldzaam moment waarbij `cursor:copy` styling relevant zou kunnen zijn ;)

Reply:
Vind ik een stap te ver gaan, zou leuk zijn maar is waarschijnlijk een hidden feature waar niemand van af weet die enkel zorgt voor extra complexie.

Reply:
Ja zou dat dan ook wel expliciet vermelden bovenaan de pagina maar 't is sowieso out of scope :)

Ik had de reflex om te denken dat dit er inzat omdat ik dat vaak zie op andere icon pages en dat ik dit intussen verwacht als ik zo'n overzicht zie maar ze kunnen natuurlijk gwn de tekst eronder copypasten.

Reply:
Voor enkel de naam van het icoon wil ik dat gedrag eventueel wel voorzien.
Dit zou geen regressie kunnen veroorzaken als bv. het vl-icon-next component verandert.